transactions/serializers.py

```diff
@@ -60,14 +60,18 @@ class Meta:
                   'price',
                   'total_amount_paid',
                   'balance', 'deposits',
-                  'percentage_completion']
+                  'percentage_completion',
+                  'image_main',
+                  'address']
 
     def get_percentage_completion(self, obj):
         """
         Return the total percentage a user has so far paid for a
         property
         """
+        print('object', obj)
         request = self.context.get('request')
+        print('request', request)
```

Comment:
Great work, I can view the property details. However, I think you forgot to remove those print you used for debugging @Jaman-dedy

Reply:
Thanks @samdeniyi , I worked on that